src/components/onboardingFormData/PersonalAndBillingDataSection.tsx

```diff
@@ -847,67 +865,69 @@ export default function PersonalAndBillingDataSection({
-                {/* Luogo di iscrizione al Registro delle Imprese facoltativo per institution Type !== 'PA' e 'PSP */}
-                <CustomTextField
-                  {...baseTextFieldProps(
-                    'businessRegisterPlace',
-                    isContractingAuthority
-                      ? t(
-                        'onboardingFormData.billingDataSection.informationCompanies.requiredCommercialRegisterNumber'
-                      )
-                      : t(
-                        'onboardingFormData.billingDataSection.informationCompanies.commercialRegisterNumber'
-                      ),
-                    600,
-                    theme.palette.text.primary
-                  )}
-                />
-              </Grid>
-              <Grid item xs={6}>
-                {/* REA facoltativo per institution Type !== 'PA' e 'PSP */}
-                <CustomTextField
-                  placeholder={'RM-123456'}
-                  {...baseTextFieldProps(
-                    'rea',
-                    t('onboardingFormData.billingDataSection.informationCompanies.rea'),
-                    600,
-                    theme.palette.text.primary
-                  )}
-                />
-              </Grid>
-              <Grid item xs={6}>
-                {/* capitale sociale facoltativo per institution Type !== 'PA' e 'PSP */}
-                <CustomTextField
-                  name={'shareCapital'}
-                  {...baseTextFieldProps(
-                    'shareCapital',
-                    isContractingAuthority
-                      ? t(
-                        'onboardingFormData.billingDataSection.informationCompanies.requiredShareCapital'
-                      )
-                      : t(
-                        'onboardingFormData.billingDataSection.informationCompanies.shareCapital'
-                      ),
-                    600,
-                    theme.palette.text.primary
-                  )}
-                  onClick={() => setShrinkRea(true)}
-                  onBlur={() => {
-                    if (!formik.values.shareCapital) {
-                      setShrinkRea(false);
-                    }
-                  }}
-                  InputLabelProps={{ shrink: shrinkRea }}
-                  InputProps={{
-                    inputComponent: NumberDecimalFormat,
-                  }}
-                />
-              </Grid>
-            </>
-          )}
+          {(isInformationCompany ||
+            isContractingAuthority ||
+            (productId === 'prod-interop' && institutionType === 'SCP')) && (
+              <>
+                <Grid item xs={12}>
+                  {/* Luogo di iscrizione al Registro delle Imprese facoltativo per institution Type !== 'PA' e 'PSP */}
+                  <CustomTextField
+                    {...baseTextFieldProps(
+                      'businessRegisterPlace',
+                      isContractingAuthority
```

Comment:
This field is mandatory for SCP that try to onboard to interop product.
The form validation and the showed label must be aligned accordingly